Week2/homework/collectiveAge.js

```diff
@@ -0,0 +1,17 @@
+const hackYourFutureMembers = [
+    { name: 'Wouter', age: 33 },
+    { name: 'Federico', age: 32 },
+    { name: 'Noer', age: 27 },
+    { name: 'Tjebbe', age: 22 },
+];
+
+function sum(item) {
+    return colAge = item.reduce((acc, curr) => acc + curr, 0);
```

Comment:
no need to assign the result to a variable. You can return the result directly. Also colAge is not declared using a var, let, const here, so ask Stelios what is the problem here. 
```suggestion
    return item.reduce((acc, curr) => acc + curr, 0);
```